packages/unmutable-core/src/CompositeMethods.js

```diff
@@ -14,60 +14,52 @@ const update = (_this: UnmutableWrapperType) => (key: *, ...args: *): Object =>
         );
 };
 
-const hasIn = (_this: UnmutableWrapperType) => (keyPath: string[]): * => {
+const hasIn = (_this: Object) => (keyPath: string[]): boolean => {
     var item = _this;
     for(let key of keyPath) {
-        if(!item.isCollection() || !item.has(key)) {
+        if(/*!item.isCollection() || */ !item.has(key)) {
```

Comment:
Should this just be removed rather than commented out? It kinda looks like it was commented out for testing.